Makefile

```diff
@@ -12,7 +12,7 @@ unit-tests:
 	python -m pytest -vv --cov=. --cov-report=$(COV_REPORT) --doctest-glob="*.md" --doctest-glob="*.rst"
 
 type-check:
-	python -m mypy .
+	python -m mypy . || x=$(python -m mypy --install-types && python -m mypy .)
```

Comment:
why? is this for local usage? you can use `make conda-env-update` to create a usable environment

Reply:
```suggestion
	python -m mypy .
```